nix/deps/nodejs-patched/default.nix

```diff
@@ -49,34 +48,15 @@ stdenv.mkDerivation {
       ${patchMavenSources} $modBuildGradle
     done
   '';
-  # Do not add a BuildId to the generated libraries, for reproducibility
-  patchBuildIdPhase = ''
-    substituteInPlace ./node_modules/react-native/ReactAndroid/src/main/jni/Application.mk --replace \
-        '-Wl,--build-id' \
-        '-Wl,--build-id=none'
-  '';
```

Comment:
Are we sure this wasn't just moved somewhere?

Reply:
Hmm yes this is on my todo list, have to check this.

Reply:
Found it `/node_modules/react-native/ReactAndroid/src/main/jni/CMakeLists.txt`

Reply:
added in latest commit.